addons/website_sale/controllers/backend.py

```diff
@@ -85,9 +85,11 @@ def fetch_dashboard_data(self, date_from, date_to):
                 ('partner_id', '!=', request.env.ref('base.public_partner').id),
                 ('invoice_status', '=', 'to invoice'),
             ]),
-            order_carts_abandoned_count=request.env['sale.order'].search_count(sale_order_domain + [
+            order_carts_abandoned_count=request.env['sale.order'].search_count([
```

Comment:
why remove sale_order_domain ? 

Now you will returl all result and not only the interval between date_from and date_to